packages/itwinui-react/src/core/ExpandableBlock/ExpandableBlock.tsx

```diff
@@ -10,28 +10,58 @@ import {
   SvgChevronRight,
   Icon,
   Box,
+  useSafeContext,
+  polymorphic,
 } from '../utils/index.js';
 import type { PolymorphicForwardRefComponent } from '../utils/index.js';
 
-type ExpandableBlockProps = {
-  /**
-   * The main text displayed on the block, regardless of state.
-   */
-  title: React.ReactNode;
-  /**
-   * Small note displayed above title, regardless of state.
-   */
-  caption?: React.ReactNode;
+const ExpandableBlockContext = React.createContext<
```

Comment:
You can reuse ExpandableBlockOwnProps to not repeat all of the already existing props.

Reply:
How do you mean?

Reply:
const ExpandableBlockContext = ExpandableBlockOwnProps & {[props that are missing from main prop type]}